humility-core/src/arch/rv.rs

```diff
@@ -36,6 +36,15 @@ impl Arch for RVArch {
         self.ei_class
     }
 
+    fn get_abi_size(&self) -> u8 {
+        match self.ei_class {
+            goblin::elf::header::ELFCLASS32 => 32,
+            goblin::elf::header::ELFCLASS64 => 64,
+            // this wont ever actually match, so we will just return 0...
```

Comment:
Why not panic? Seems unlikely that you'll be able to continue in a meaningful manner assuming registers are 0-bits wide.

Reply:
Correct, it would probably result in nothing being printed.  I am going to change to an `unimplemented` macro.